test-data/unit/check-based-incomplete-defs.test

```diff
@@ -31,3 +31,16 @@ allow_incomplete_defs = True
 allow_untyped_defs = True
 \[mypy-b]
 incomplete_is_typed = True
+
+
+[case typeIgnoreWorksProperly]
+# flags: --python-version 3.8
+# This test is to check that the `notes` param on `mypy.errors.Errors.report` works correctly
+def f(a, b: int = 1): ...  # type: ignore[no-untyped-def]
+f(
+1)  # type: ignore[no-untyped-call]
+[out]
+main:4: error: Call to incomplete function "f" in typed context  [no-untyped-call]
+main:4: note: Type is "def (a: Untyped, b: int =) -> None"
+main:5: error: Unused "type: ignore" comment  [unused-ignore]
+[out version>=3.8]
```

Comment:
what about scenarios where the `type:ignore` works even tho its on the next line

Reply:
That is this scenario.

Reply:
why the `unused-ignore` error then?

Reply:
3.7 doesn't support ignores on the wrong line, so it's expecting errors. But the `[out version>=3.8]` check is expecting no outputs.